modules/ROOT/pages/functions/dev_guide/internal_registry/using-an-internal-registry.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies
+//
+// functions/dev_guide/internal_registry/internal-registry.adoc
+
+A common use case for building and deploying Functions is to use an external container image registry, such as Quay. However, you can instead use an OpenShift Container Platform internal registry, if it has been exposed:
```

Comment:
Minor nit
```suggestion
When building and deploying functions, the resulting container image is stored in an image registry. In many cases, this will be a public registry, such as Quay. However, you can instead use an OpenShift Container Platform internal registry, if it has been exposed:
```